### ISSUE TITLE
Sound effects volume is sometimes too low

### ISSUE THREAD

**Issue by potsmugen** [ikemen-engine/Ikemen-GO#1183]
### Describe the bug

As pointed out by several people on Discord, there are some issues with sound volume in the latest build. At least with character sound effects. Some sounds will be a lot softer than they should, others may even be inaudible. Seems related to their sample rate and such factors.

This issue is either less pronounced or not present in 0.98.2.

### To Reproduce

[One such character is this one.](https://mugenguild.com/forum/topics/i-release-poison-kof-ver-197657.0.html)

Sounds will play correctly in Mugen 1.1 and 0.98.2, but not latest build.

### Expected behavior

Same as Mugen 1.1?

### Screenshots / Video

See replies.

### Engine Version (or source code date)

22/04/2023 as well as 0.99 RC1

### Operating system

Windows

**Comment by Eiton** [comment]
Looks like the problem of the linked character is about sound channel.
Currently Ikemen Go picks the largest available channel number when channel is not specified in playsnd. At default setting it would pick 31 when all the channels are available.

There is a stopsnd on statedef -2 of the character. The sounds play normally if you remove it.
```
[State 100, 3]
type = StopSnd
trigger1 = stateno != 100
channel = 31
```

**Comment by potsmugen** [comment]
Interesting. Then I guess that character is the wrong test case for this issue as they are different issues.

Here's a better example of the volume issues:

https://github.com/ikemen-engine/Ikemen-GO/assets/107247004/a3073797-579b-4470-ad66-289e5cd01752

**Comment by K4thos** [comment]
fixed by #1389
